src/main/resources/db/migration/V1_29__insert_judicial_location_mapping.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO judicial_location_mapping (epimms_id, judicial_base_location_id, building_location_name, base_location_name, service_code) VALUES('', '1032', '', 'Social Entitlement', 'BBA3');
+DELETE FROM judicial_service_code_mapping WHERE service_id ='21' OR service_id ='55';
```

Comment:
Please use the following query for delete:

DELETE FROM judicial_service_code_mapping WHERE service_id  in ('21', '55');

Reply:
change is done and updated PR , kindly reivew